Sources/MSLCombine/PublisherConvertible.swift

```diff
@@ -0,0 +1,23 @@
+import Combine
+
+/**
+ PublisherConvertible allows us to mark a variable as @Published without having to rely on SwiftUI's implementation.
+ This will expose a Publisher.
```

Comment:
Would we add an "Example" here to show how this helps?

Reply:
I also think the comment might be incorrect? I think you have to mark it with `@PublisherConvertible` right? Not `@Published`?

Reply:
I think the intention of the comment is that this is equivalent to marking something as Published without using the SwiftUI annotation. I'll add in an example!

Reply:
Oh, I forgot I already added one :joy:

It's in `PublisherConvertibleExample.swift`

Reply:
(Let me know if there's anything else I can add to that example!)

Reply:
Two thoughts here:

Although this custom property wrapper may be _similar_ to a `@Published` from SwiftUI, I think it's use cases are not necessarily the same. Instead of saying this is like something else (which would require the reader to have knowledge of the other thing), I might recommend just describing what this does. Maybe something like this?

```
@PublisherConvertible provides a convenient way to build a publisher around a specific property. This is particularly useful when you want to subscribe to the changing of the property's values.
```

My second thought is around examples. Personally I've always liked the examples right inside the comment. This way, it's a bit easier to find, understand, and then implement. I thought Jeff did a really good job when he built the `Coordinator`.

<img width="1072" alt="image" src="https://github.com/MichiganLabs/MSLToolkit/assets/4023203/e72fa701-3a86-48b6-90d2-8b481699be74">

My fear with having an Example folder is that maybe people won't know where to look, or struggle finding an example if the file name isn't what they expect, or find nothing at all if we weren't diligent in adding an example file for each utility that we make.

But maybe there is a simple way to link the examples you created to the implementation?